t/57-oracle-datetime.t

```diff
@@ -30,7 +30,8 @@ without $dbh {
 ok $dbh,    'Connected';
 my $dropper = q|
     BEGIN
-       EXECUTE IMMEDIATE 'DROP TABLE test_datetime';
+       EXECUTE IMMEDIATE 'PURGE RECYCLEBIN';
```

Comment:
Why did you add this statement? In a shared test database it might have unwanted side effects.

Reply:
It's called cleaning up after yourself; This test left unchecked (I'm sure other test scripts are dirty as well) will waste resources every time a table is created and dropped. Its my experience that folks will go unaware that the RECYCELEBIN is filling up or that it even exists. It will produce no negative issues. Anyone relying on the RECYCLEBIN as a long time storage recovery of discarded tables is looking for trouble. We're doing folks a favor.